Registry/Registry.EM_COMMON

```diff
@@ -2140,12 +2140,12 @@ rconfig   integer     bl_mynn_tkebudget   namelist,physics      max_domains    0
 rconfig   integer     ysu_topdown_pblmix  namelist,physics      1              0       rh       "ysu_topdown_pblmix"            ""      ""
 rconfig   integer     shinhong_tke_diag   namelist,physics      max_domains    0       rh       "shinhong_tke_diag"             ""      ""
 rconfig   logical     bl_mynn_tkeadvect   namelist,physics      max_domains    .false.  rh      "bl_mynn_tkeadvect"             ""      ""
-rconfig   integer     bl_mynn_cloudpdf    namelist,physics      1              2       irh      "bl_mynn_cloudpdf"              ""      ""
-rconfig   integer     bl_mynn_mixlength   namelist,physics      1              2       irh      "bl_mynn_mixlength"             "0:original,1:RAP/HRRR,2:new blending&cloud mix length" ""
+rconfig   integer     bl_mynn_cloudpdf    namelist,physics      1              2       irh      "bl_mynn_cloudpdf"              "0:original, 1:Kuwano, 2:Chaboreau-Bechtold"            ""
+rconfig   integer     bl_mynn_mixlength   namelist,physics      1              1       irh      "bl_mynn_mixlength"             "0:original,1:RAP/HRRR,2:new blending&cloud mix length" ""
 rconfig   integer     bl_mynn_edmf        namelist,physics      max_domains    1       irh      "bl_mynn_edmf"                  "0:off,1:activate mass-flux in mynn"                    ""
-rconfig   integer     bl_mynn_edmf_mom    namelist,physics      max_domains    0       irh      "bl_mynn_edmf_mom"              "0:off,1:activate mass-flux transport of momentum"      ""
+rconfig   integer     bl_mynn_edmf_mom    namelist,physics      max_domains    1       irh      "bl_mynn_edmf_mom"              "0:off,1:activate mass-flux transport of momentum"      ""
 rconfig   integer     bl_mynn_edmf_tke    namelist,physics      max_domains    0       irh      "bl_mynn_edmf_tke"              "0:off,1:activate mass-flux transport of tke"           ""
-rconfig   integer     bl_mynn_edmf_part   namelist,physics      max_domains    0       irh      "bl_mynn_edmf_part"             "0:off,1:activate areal partitioning of ed and mf"      ""
+rconfig   integer     bl_mynn_mixscalars  namelist,physics      max_domains    0       irh      "bl_mynn_mixscalars"            "0:off,1:activate mixing of scalars (qnx, qnxfa) in MYNN"      ""
```

Comment:
Can we assume that no one else would have set bl_mynn_edmf_part in a namelist? If this is largely an "internal developer" switch that nobody else would know about or use, then I am OK with the re-name (without some version overlap and helpful comments in the check_a_mundo).

Reply:
Yes, or if they did, the users were wasting their time. The flag bl_mynn_edmf_part partitioned the eddy diffusivity (ED) and mass-flux (MF) by reducing the ED when the MF scheme was active. This was done by assuming there is no ED in grid cell where the MF is active. This is a poor assumption, but it had very little or no impact, so as I saw it, it was a waste of electricity and needed to be removed.

Thus, bl_mynn_edmf_part was replaced with bl_mynn_mixscalars, which serves a completely different function: to mix the second moments and aerosols within the MYNN. This makes the MYNN portable to other models (like FV3) because it doesn't have to rely upon the code associated with the scalar_pblmix option in WRF.

I will add more detail in the commit description to clarify this.